<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clsCalibration</class>
 <widget class="QDialog" name="clsCalibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>多通道扫描箱校准</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/res.qrc">
    <normaloff>:/lightWave.png</normaloff>:/lightWave.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">clsCalibration
{
 background-color: rgb(144, 148, 248);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>多通道校准</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="btnOpenTrim">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>开路
校准</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lblOpenTrim">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btnShortTrim">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>短路
校准</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lblShortTrim">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>多通道负载校准</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,1">
        <item>
         <widget class="QPushButton" name="btnGoldenCalibrate">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>标样
校准</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblGoldenSample">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>消息显示</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="lblHFTrim_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>校准类型</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblHFTrim_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>校准频率</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lblHFTrim_5">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>校准通道</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblCalbrationType">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lblCalFrequency">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lblCalChannel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="ClickedLabel" name="lblswtich">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMultiFreq">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>频率点
文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClearCalData">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>清除
校准</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDone">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>完成</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickedLabel</class>
   <extends>QLabel</extends>
   <header>ClickedLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/res.qrc"/>
  <include location="../images/res.qrc"/>
  <include location="../images/res.qrc"/>
 </resources>
 <connections/>
</ui>
